zookeeper-server/src/test/java/org/apache/zookeeper/server/CRCTest.java

```diff
@@ -102,8 +103,7 @@ private boolean getCheckSum(File snapFile) throws IOException {
      * @throws Exception
      */
     @Test
-    public void testChecksums() throws Exception {
-        File tmpDir = ClientBase.createTmpDir();
+    public void testChecksums(@TempDir File tmpDir) throws Exception {
```

Comment:
I did not know you could do this! That's neat.

Reply:
This method has some issue on Windows but since we tests on *nix it should be fine.

See https://github.com/junit-team/junit5/issues/2811 and so on.